packages/opentelemetry-context-base/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Same here? It would be simpler to maintain one common file instead of having same boilerplate code in each package.

Reply:
It's not always the same since some packages are shared for the web and need some changes.

Reply:
I can check if we can centralize a bit more...